Editors Update - Adding additional troubleshooting steps for Visual Studio Code

getHashSet opened the pull request:
Additional Troubleshooting steps added for Visual Studio Code users that are unable to grant $PATH access to their VSCode instances.

Additional steps to resolve include opening a new instance of VSCode through the built in terminal windows.

Added code examples and testing notes.

getHashSet committed:
Additional Troubleshooting Steps Added

Expanded on troubleshooting steps for Ubuntu shells that do not adopt the $PATH when launching `code .` from the command line.

getHashSet committed:
Tested in VM. Added requirements section.

The Visual Studio Code editors guide now has prerequisites added to "Additional Troubleshooting Steps" section

rikgirbes commented on book/docs/setup/editors/vscode.md:
```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
+### Possible Solution
+1. Open the built-in terminal inside your Visual Studio Code by pressing `control` _+_ `~ ` at the same time or by going to `Terminal > New Terminal` within the file menu.
+
+2. From the built in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```
```suggestion
2. From the built-in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```

Keeping consistent with 1.

getHashSet replied:
I totally agree! Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

rikgirbes commented on book/docs/setup/editors/vscode.md:
````diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
+```
+
+### How to Check
+_Select a function in the Visual Studio Code Editor and right click and choose **Go to Definition** or highlight a function and press **F12**_.
````
```suggestion
_Select a function in the Visual Studio Code Editor and right-click and choose **Go to Definition** or highlight a function and press **F12**_.
```

getHashSet replied:
Good catch. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

rikgirbes commented on book/docs/setup/editors/vscode.md:
```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
+If everything is fine you should get auto-completion and other features working 🎉
+
+---
+</br>
```
</br> redundant

getHashSet replied:
Bad habits die hard. I have removed the `</br>` line break. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

rikgirbes reviewed (changes requested):
VERY little feedback 😊 

Otherwise: Awesome, thanks! 👍🏽

getHashSet committed:
Punctuation updates

Hyphens added to document to prevent distracting punctuation while triaging $PATH issues.
